bpf/unwinders/native.bpf.c

```diff
@@ -948,6 +948,7 @@ int native_unwind(struct bpf_perf_event_data *ctx) {
             LOG("[warn] mapping not found");
             request_refresh_process_info(ctx, per_process_id);
             BUMP_UNWIND_FAILED_COUNT(per_process_id, mapping_not_found);
```

Comment:
We can probably get rid of `BUMP_UNWIND_FAILED_COUNT`, `failed_unwind_reasons`, etc. now

Reply:
I was thinking these could still be useful for surveillance purposes (ie if we got them exported into prometheus).

Reply:
We already have a bunch of Prometheus metrics from the unwinder, which doesn't work via this `BUMP_UNWIND_FAILED_COUNT` mechanism. See e.g. `unwinder_stats_t` and the `DEFINE_COUNTER` mechanism.